test/langtools/jdk/javadoc/doclet/testClassTree/badpkg/ChildClass.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
These days, the preferred style is to write small files on the fly during the test.  That way, the test class is better co-located with the the code that uses it, and as a secondary benefit, you do not need to include a full legal header that tends to overwhelm the few lines of interest.